src/components/OpenWith/ExecuteForm.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @flow
+ * @file Form to invoke an integration via POST
+ * @author Box
+ */
+
+import * as React from 'react';
+
+type Props = {
+    formData: ExecuteAPI,
+    id: string,
+};
+
+const ExecuteForm = ({ formData: { url, params }, id }: Props) => (
+    <React.Fragment>
+        <form
+            id={`bcow-execute-form-${id}`}
+            action={url}
+            target="_blank"
+            acceptCharset="utf-8"
+            method="post"
+            encType="multipart/form-data"
```

Comment:
Form defaults are application/x-www-form-urlencoded
multipart/form-data is mostly for binary
https://stackoverflow.com/questions/4007969/application-x-www-form-urlencoded-or-multipart-form-data

Also,